index.html

```diff
@@ -40,7 +40,7 @@
             <noscript>
                 <div id="message">
                     <img src="logo.png" alt="Rose Vouchers">
-                    <p>It appears JavaScript is currently disabled in your browser. Please re-enable it to correctly display the app. Thank you for using Rose Vouchers.</p>
+                    <p>It appears JavaScript is currently disabled in your browser. Please re-enable it to correctly display the app. Thank you for using Rosie - Rose Voucher Records &amp; Reimbursement.</p>
```

Comment:
Is  '..."Rosie",  the Rose Voucher Records &amp; Reimbursement app' more friendly?